libs/application/templates/public-debt-payment-plan/src/fields/components/Slider/Slider.tsx

```diff
@@ -118,11 +123,16 @@ const Slider = ({
   const dragBind = useDrag({
     onDragMove(deltaX) {
       const currentX = x + deltaX
+      const roundedMin = toFixedNumber(min, 1, 10)
       dragX.current = Math.max(0, Math.min(size.width, currentX))
       const index =
         dragX.current / sizePerCell + min >= max
           ? max
-          : roundByNum(dragX.current / sizePerCell, step) + min
+          : dragX.current / sizePerCell + min <= min
+          ? min
+          : roundByNum(dragX.current / sizePerCell, step) === 0
+          ? min
+          : roundByNum(dragX.current / sizePerCell, step) + roundedMin
```

Comment:
perhaps add a comment explaining what is going on here, I feel like this will be hard to understand when coming back to it later

Reply:
Done